src/components/ui/select.tsx

```
@@ -0,0 +1,177 @@
+'use client'
+
+import * as SelectPrimitive from '@radix-ui/react-select'
+import { CheckIcon, ChevronDownIcon, ChevronUpIcon } from 'lucide-react'
+import type { ComponentProps } from 'react'
+import { cn } from '~/lib/utils'
+
+import type { ComponentProps } from 'react'
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ComponentProps'. Consider to delete it or rename it.

'ComponentProps' is defined here:


(lint/suspicious/noRedeclare)